### PR TITLE
[feat] Add traces, now served as submodule to zkevm repo.

### PR DESCRIPTION
**Pull request by 29988122**
https://github.com/scroll-tech/scroll-zkevm/pull/114

Todo: Work w/ @ChuhanJin to add the rest bridge related traces for ERC20/native deposit, withdraw and approve from L1/L2.

### CODE REVIEW

**Comment by 0xmountaintop** [comment]
we have both multiple.json & mutiple.json?

**Comment by 29988122** [comment]
Got a typo when replacing file. Fixed!

**Comment by 29988122** [comment]
Updated, now generated from scroll-v3.1.12. 
Todos: Update Scroll bridge related traces.

**Comment by 29988122** [comment]
Todo: autoci is wrongfully reverting changes in 03.json and 19.json.

**Comment by lispc** [comment]
need to be updated with l2geth HEAD

**Comment by 0xmountaintop** [comment]
> Todo: autoci is wrongfully reverting changes in 03.json and 19.json.

CI doesn;t revert 03.json/19.json. if you check, the content/version is the same as your commit.

The CI failing is not due to 03.json/19.json, but `bridge/withdrawETH.json` & `bridge/approveUSDC.json`. see https://results.pre-commit.ci/run/github/557658762/1684498755.tBkIgvLSRJmowsR2vKZGSg

**Review by lispc — APPROVED** [review]

File: bridge/approveUSDC.json
Comment (0xmountaintop) [inline bridge/approveUSDC.json:1]: why not delete these if they are not ready?

File: bridge/depositETH.json
Comment (0xmountaintop) [inline bridge/depositETH.json:1]: ditto

File: bridge/depositUSDC.json
Comment (0xmountaintop) [inline bridge/depositUSDC.json:1]: ditto

File: bridge/swap/AddLiquidity.json
Comment (0xmountaintop) [inline bridge/swap/AddLiquidity.json:1]: ditto

File: bridge/swap/Approve_initBalance.json
Comment (0xmountaintop) [inline bridge/swap/Approve_initBalance.json:1]: ditto

File: bridge/swap/Swap.json
Comment (0xmountaintop) [inline bridge/swap/Swap.json:1]: ditto

File: bridge/withdrawETH.json
Comment (0xmountaintop) [inline bridge/withdrawETH.json:1]: ditto

File: bridge/withdrawUSDC.json
Comment (0xmountaintop) [inline bridge/withdrawUSDC.json:1]: ditto

**Review by 0xmountaintop — APPROVED** [review]

**Pull request merged**